sectionsUtilityTool/src/main/webapp/assets/js/utils.js

```diff
@@ -85,6 +85,25 @@ var utilPopWindow = function(url, name){
     return false;
 };
 
+var doXListPosts = function(posts){
+  var index, len;
+  var xListPosts = [];
+  for (index = 0, len = posts.length; index < len; ++index) {
+    var xListUrl ='manager' + posts[index];    
+    xListPosts.push(
+      $.post(xListUrl, function(data) {
+        var section = data.id;
+        $('#xListSection' +  section).append(' <span class=\"label label-success\">Success</span>');
+      })
+      .fail(function(data) {
+        var section = data.id;
+        $('#xListSection' +  section).append(' <span class=\"label label-failure\">Failure</span>');
```

Comment:
From the viewpoint of the end user, I'm not sure if 'Failure' is enough information. May want to change it to something like 'Failure to POST' or something with more information.